server/base/src/main/java/org/apache/accumulo/server/util/Admin.java

```diff
@@ -162,6 +162,13 @@ static class RandomizeVolumesCommand {
     String tableName = null;
   }
 
+  @Parameters(commandDescription = "Verify tablet assignments")
```

Comment:
Description could be better... What about "Verify all Tablets are assigned to tablet servers"

Reply:
@milleruntime - This has been updated